src/Command/DoctrineDecryptDatabaseCommand.php

```diff
@@ -146,5 +146,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln('' . PHP_EOL . 'Decryption finished values found: <info>' . $valueCounter . '</info>, decrypted: <info>' . $this->subscriber->decryptCounter . '</info>.' . PHP_EOL . 'All values are now decrypted.');
+        return 1;
```

Comment:
You should return 0 if the command finished successfully.  This is in line with how every command in linux works.

Also see the DocBloc of `Symfony\Component\Console\Command::execute`: 
> @return int 0 if everything went fine, or an exit code

This applies to the other commands as well.

Reply:
Agree. You can avoid ambiguity with the equivalent:

`return Command::SUCCESS`

Reply:
That only works for symfony 5.1 as far as I can see.  So I'd rather not use those constants for now since older versions of symfony are supported at this moment.

Reply:
So it's fine like it is, isn't it?

I mean:
This PR looks good to me.:)

Reply:
I disagree that it looks fine.  I believe every return-statement added in this MR should return `0`, which is the generally accepted value for success.

Returning `1`, like it is now, would signal that something went wrong.

The correct statement would be:
```php
return 0;  // To be replaced by self::SUCCESS whenever we upgrade minimum-requirements of symfony/console to 5.1
```

Reply:
Ahh yes! You are right!